lib/matplotlib/pyplot.py

```diff
@@ -1997,6 +1997,49 @@ def colormaps():
     return sorted(cm.cmap_d)
 
 
+def verbose(level='info'):
+    """
+    Shortcut to set the debugging level of the logging module.
+
+    Parameters
+    ----------
+    level : str or bool (optional)
+        If *True* (default) set logging level to "INFO" for matplotlib module.
+        If *False* set logging level to "WARNING" for matplotlib module (this
+        is the default level if ``verbose`` is not called). If a string, must
+        be one of ['warning', 'info', 'debug'], in order of increasing
+        verbosity.
+
+    Notes
+    -----
+    This is the same as the standard python `logging` module::
+
+        import logging
+
+        _log = logging.getLogger(__name__)
+        _log.setLevel(logging.DEBUG)  # or logging.WARNING, logging.INFO
+
+    """
+    matplotlib.verbose(level=level)
+
+
+def debug():
```

Comment:
The idea here is to tell the user to do one thing and `plt.debug()` is (a bit) easier than `plt.verbose('debgug')`.  Originally I wanted `plt.verbose` to go to `debug`, but @tacaswell thought it should just be `info`.

Reply:
Simplifying `plt.verbose('debug')` to `plt.debug()` is unnecessary. Our users know how to pass a parameter. I'm afraid it would even be more confusing. As a user, I would ask myself: Is there a difference between those two? There should be only one function for setting the log level. 

I also recommend naming the function `plt.set_loglevel()`, which is more in line with the terminology of the `logging` module. "verbose" was coming from the command line interface. In this world it as a well-known meaning. Not so in Python code. Here, the well-known entity for controlling output is the log level. Since Matplotlib is a library we should stick to that terminology.